<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QWidget" name="ManagerWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>酒店管理员</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="login">
    <widget class="QLabel" name="lb_adminname">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>管理员用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_adminpswd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>管理员密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_adminname">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;
color: #000000;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_adminpswd">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;
color: #000000;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_signIn">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>239</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_verdict">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ff0000;</string>
     </property>
     <property name="text">
      <string>管理员用户名或密码错误</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="managerFunc">
    <widget class="QPushButton" name="bt_back">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_checkReport">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查看报表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_refresh">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lw_roomState">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>391</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>房间信息</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
